### DISCUSSION TITLE
Add zellij profile

### DISCUSSION THREAD

**Discussion by noahbald** [ibhagwan/fzf-lua#1570] [Ideas]
Hi, just started using this plugin! I think it would be nice if there was a profile for running fzf commands in a floating zellij window, similar to how I assume there is for tmux.

**Comment by ibhagwan** [comment]
> Hi, just started using this plugin! I think it would be nice if there was a profile for running fzf commands in a floating zellij window, similar to how I assume there is for tmux.

These are controlled by the fzf/skim binaries with `fzf --tmux`, although I could work my way around it I believe the upstream is better suited to add such functionality properly.

**Reply by noahbald** [reply]
I started a discussion in fzf (https://github.com/junegunn/fzf/discussions/4134) and it looks like using `fzf --tmux` or `fzf --popup` will end up opening a zellij window out of the box in a coming PR.
So once this is in, hopefully no changes in fzf-lua are required.

**Reply by ibhagwan** [reply]
> fzf --popup

This will require a minor change to make sure a neovim window doesn’t open when using the new flag.
